llvm/test/CodeGen/X86/machine-function-cfg-hash.ll

```diff
@@ -0,0 +1,81 @@
+; RUN: llc < %s -O0 -mtriple=x86_64-pc-linux -function-sections -basic-block-sections=labels \
```

Comment:
Maybe we don't need to add and test the dump option.

Reply:
Done